movies-app/src/components/MovieModal/MovieModal.js

```diff
@@ -0,0 +1,31 @@
+import { Portal } from "react-portal";
+import { FocusTrap } from 'focus-trap-react';
+import MovieForm from "../MovieForm/MovieForm";
+import './MovieModal.css'
+
+const MovieModal = ({ isModalOpen, modalHeader, movieInfo, onClose }) => {
+    const onModalClose = () => onClose(false);
+    return (
+        isModalOpen && <Portal>
+            <div className={`modal fade ${isModalOpen ? 'show' : ''}`} id="exampleModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true" style={{ display: isModalOpen ? 'block' : 'none' }}>
+                <div className="modal-dialog modal-dialog-centered modal-lg dark-modal">
+                    <div className="modal-content">
+                        <div className="modal-header">
+                            <h1 className="modal-title fs-5" id="exampleModalLabel">{modalHeader}</h1>
+                            <button type="button" className="btn-close" onClick={onModalClose} data-bs-dismiss="modal" aria-label="Close">
+                                <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='%23000'>
```

Comment:
can we put the svg's in the different folder so that it can be reused ?

Reply:
Moved to assets folder